packages/viewer/src/modules/extensions/measurements/MeasurementsExtension.ts

```diff
@@ -408,13 +409,26 @@ export class MeasurementsExtension extends Extension {
     }
   }
 
-  public removeMeasurement() {
-    if (this._selectedMeasurement) {
-      this._measurements.splice(
-        this._measurements.indexOf(this._selectedMeasurement),
-        1
-      )
-      this.renderer.scene.remove(this._selectedMeasurement)
+  private measurementDataId(measurementData: MeasurementData) {
```

Comment:
JSON.stringify to get id is not ideal, but seems to be the best option

Reply:
I don't think it is practical. Why don't we just generate an uuid when creating them, and add an `id` filed in the data model?

Reply:
the benefit of JSON.stringify is that if two measurements are structurally identical, they are treated as identical. I was concerned about the same measurement somehow arriving w/ multiple uuids because they could get re-generated, but I think you're right and it should be possible to avoid this